test/stablecoin_converter.js

```diff
@@ -173,6 +174,45 @@ contract("StablecoinConverter", async (accounts) => {
 
       await truffleAssert.reverts(stablecoinConverter.addToken((await ERC20.new()).address), "Max tokens reached")
     })
+
+    it("Burn 10 OWL on Add Token", async () => {
+      const TokenOWLProxy = artifacts.require("../node_modules/@gnosis.pm/owl-token/build/contracts/TokenOWLProxy")
+      const owlToken = await TokenOWL.new()
+      const owlProxyContract = await TokenOWLProxy.new(owlToken.address)
+      const owlProxy = await TokenOWL.at(owlProxyContract.address)
+      await owlProxy.setMinter(user_1)
+      const owlAmount = (10 * 10 ** 18).toString()
```

Comment:
Similar comment as before: is there value in doing `10 * 10 ** (await owlProxy.decimals())` here? If the value ends up being hard coded in the constructor this would be a good sanity check that the amount is what you expect it to be.